src/components/auth/login/LoginPage.css

```diff
@@ -0,0 +1,46 @@
+:root {
+    --maroon: #602040;
+
+}
+.login{
+    text-align: center;
+    margin: 5em;
+}
+.login > h1{
```

Comment:
Ako ces ovaj h1 primenjivati na citavom APP, onda bi to mogla na globalnom nivou da setujes, da se odnosi na sve h1, isto moze i za  font-family.

ovako nesto mislim :
```
*{
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

body{
  font-family: Arial, Helvetica, sans-serif;
  background-color: #fff;
  color: #000;
  font-size: 15px;
  line-height: 1.5;
}
h1{
  color:var(--maroon)
}
```